inlong-audit/audit-store/src/main/java/org/apache/inlong/audit/service/ElasticsearchService.java

```diff
@@ -60,19 +59,29 @@
 public class ElasticsearchService implements InsertData, AutoCloseable {
 
     private static final Logger LOG = LoggerFactory.getLogger(ElasticsearchService.class);
+    private static final Gson GSON;
```

Comment:
The issue associated with this PR describes an issue with the ClickHouse SQL files, and it is not recommended to submit other unrelated codes in this PR.

Reply:
OK, I will remove them.